server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -296,6 +307,19 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
 		return scribeLambda;
 	}
 
+    private async getOpMessages(documentId: string, tenantId: string, lastCheckpoint: IScribe): Promise<ISequencedDocumentMessage[]>{
+        let opMessages: ISequencedDocumentMessage[] = [];
+        if (!this.getDeltasViaAlfred) {
+            // Fetch pending ops from scribeDeltas collection
+            const dbMessages =
+                await this.messageCollection.find({ documentId, tenantId }, { "operation.sequenceNumber": 1 });
```

Comment:
@pradeepvairamani  I thought we are not reading from scribeDeltas anymore here, right?

Reply:
That is correct..this is behind the !getDeltasViaAlfred condition, which is there just in case we want to fall back to the old way of doing things. Once it is well validated, the plan is to get rid of this code along with the flag